polymesh_schema.json

```
@@ -1356,6 +1334,27 @@
           }
         ],
         "type": "Option<KeyIdentityData>"
+      },
+      "validCDDClaims": {
+        "description": "Returns all valid IdentityClaim of type CustomerDueDiligence for the given target_identity",
+        "params": [
+          {
+            "name": "target_identity",
+            "type": "IdentityId",
+            "isOptional": false
+          },
+          {
+            "name": "cdd_checker_leeway",
+            "type": "u64",
+            "isOptional": false
```

This one can use the `isOptional: true` feature, since it is at the end.
```suggestion
            "isOptional": true
```